packages/playwright-test/types/testExpect.d.ts

```diff
@@ -48,18 +42,34 @@ export declare type Expect = {
 
 type Awaited<T> = T extends PromiseLike<infer U> ? U : T;
 
-type OverriddenExpectProperties =
-  'not' |
-  'resolves' |
-  'rejects' |
-  'toMatchInlineSnapshot' |
-  'toThrowErrorMatchingInlineSnapshot' |
-  'toMatchSnapshot' |
-  'toThrowErrorMatchingSnapshot';
+type SupportedExpectProperties =
+  'toBe' |
+  'toBeCloseTo' |
+  'toBeDefined' |
+  'toBeFalsy' |
+  'toBeGreaterThan' |
+  'toBeGreaterThanOrEqual' |
+  'toBeInstanceOf' |
+  'toBeLessThan' |
+  'toBeLessThanOrEqual' |
+  'toBeNaN' |
+  'toBeNull' |
+  'toBeTruthy' |
+  'toBeUndefined' |
+  'toContain' |
+  'toContainEqual' |
+  'toEqual' |
+  'toHaveLength' |
+  'toHaveProperty' |
+  'toMatch' |
+  'toMatchObject' |
+  'toStrictEqual' |
+  'toThrow' |
+  'toThrowError'
 
 declare global {
   export namespace PlaywrightTest {
-    export interface Matchers<R> extends Omit<expect.Matchers<R>, OverriddenExpectProperties> {
+    export interface Matchers<R> extends Pick<expect.Matchers<R>, SupportedExpectProperties> {
```

Comment:
Do we have an idea on how many users we break and how? Should this have a large 'Breaking changes' section somewhere?

Reply:
It was never working from the implementation side (no access to the jest.fn() function), so it should be fine.

Reply:
Let's get ourselves a list in the comments in this file where we explain why we ban everything else!